editor.planx.uk/src/pages/FlowEditor/components/Sidebar/Search/facets.ts

```diff
@@ -9,7 +9,6 @@ const generalData: SearchFacets = ["data.fn", "data.val"];
 const fileUploadAndLabelData: SearchFacets = ["data.fileTypes.fn"];
 
 const calculateData: SearchFacets = [
-  "data.output",
```

Comment:
This is another key benefit - `data.output` is now simply picked up by `generalData` search facets! :heavy_check_mark: